src/photomosaic.js

```diff
@@ -113,19 +113,38 @@
        */
       PhotoMosaic.prototype.tileCanvas = function(context) {
           var processedCanvas = document.createElement('canvas');
-          processedCanvas.width = context.canvas.width;
+          var width = processedCanvas.width = context.canvas.width;
           processedCanvas.height = context.canvas.height;
 
           var processedContext = processedCanvas.getContext('2d');
 
           var options = this.options;
+          
+          var originalImageData = context.getImageData(0, 0, context.canvas.width, context.canvas.height);
+
+          function getImageData(startX, startY, tileWidth, tileHeight) {
```

Comment:
Can we change the function name as it confuses with `context.getImageData()` and move it outside this function ?